libs/events/import/api/jest.config.ts

```diff
@@ -2,14 +2,15 @@
 export default {
   displayName: 'events/import-api',
   preset: '../../../../jest.preset.js',
-  globals: {
```

Comment:
This way was deprecated, I guess this library was generated with an older nx version.